ShopGo/LoginScreen/LoginVC.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  LoginVC.swift
+//  GoToShop
+//
+//  Created by Waqas Shabir on 7/23/23.
+//
+
+import UIKit
+class LoginVC: UIViewController {
+
+    @IBOutlet weak var logBtnLabel: UIButton!
+    @IBOutlet weak var userNameTxt: UITextField!
+    @IBOutlet weak var passwordTxt: UITextField!
```

Comment:
fix the naming of these outlets

Reply:
1: loginButton
2: userNameTextField 
3: passwordTextField